dev-packages/e2e-tests/test-applications/nextjs-app-dir/tests/client-app-routing-instrumentation.test.ts

```diff
@@ -39,7 +39,10 @@ test('Creates a navigation transaction for app router routes', async ({ page })
 
   const serverComponentTransactionPromise = waitForTransaction('nextjs-app-dir', async transactionEvent => {
     return (
-      transactionEvent?.transaction === 'GET /server-component/parameter/foo/bar/baz' &&
+      // It seems to differ between Next.js versions whether the route is parameterized or not
```

Comment:
This seems bad and something we should fix. Doesn't this mean paramaterization breaks in the newest nextjs version?